Dockerfile

```diff
@@ -1,17 +1,17 @@
-FROM node:4.2
+FROM node:10.6.0-alpine
 
-RUN npm install -g -q protractor
+RUN apk update && apk upgrade && \
+    apk add --no-cache git curl yarn 
 
 WORKDIR /usr/src/app
 
-COPY package.json bower.json .bowerrc .npmrc ./
+COPY package.json .npmrc ./
 
-RUN npm install -q
+RUN yarn
```

Comment:
we should use yarn install --flat, yarn install --production

Reply:
yes will do